src/admin/components/utilities/Auth/index.tsx

```diff
@@ -44,29 +45,56 @@ export const AuthProvider: React.FC<{ children: React.ReactNode }> = ({ children
 
   const id = user?.id;
 
-  const refreshCookie = useCallback(() => {
+  const refreshCookie = useCallback((forceRefresh?: boolean) => {
```

Comment:
I don't see anywhere in our code where we call refreshCookie with forceRefresh. Is this being used?